hw_steering_src/dr_definer.py

```diff
@@ -95,11 +85,7 @@ def definer_dws_parser(self):
         for i in range(6):
```

Comment:
I think we need to fix all magic numbers, lets define them above or in separate file

Reply:
range(6), 6 is the number of dws in a definer?

Reply:
Done